frontend/src/pages/projects/screens/detail/const.ts

```diff
@@ -1,7 +1,17 @@
-import { ProjectSectionID, ProjectSectionTitlesType } from './types';
+import {
+  ProjectSectionID,
+  ProjectSectionIDExtended,
+  ProjectSectionTitlesType,
+  ProjectSectionTitlesTypeExtended,
+} from './types';
 
 export const ProjectSectionTitles: ProjectSectionTitlesType = {
   [ProjectSectionID.WORKSPACE]: 'Data science workspaces',
   [ProjectSectionID.STORAGE]: 'Storage',
   [ProjectSectionID.DATA_CONNECTIONS]: 'Data connections',
 };
+
+export const ProjectSectionTitlesExtended: ProjectSectionTitlesTypeExtended = {
+  ...ProjectSectionTitles,
+  [ProjectSectionIDExtended.MODEL_SERVER]: 'Model server',
```

Comment:
Yeah, definitely thinking we should just merge this together with the previous const of labels.